pom.xml

```diff
@@ -319,6 +319,12 @@
         <artifactId>beam-sdks-common-fn-api</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
```

Comment:
I'll field this one - they are actually entirely different dependencies - one on the tests and one on the main code. And the `<dependencyManagement>` block doesn't actually set up any dependencies, it just sets up the version that will be used by default if it is omitted in some other module's `<dependencies>` block.